src/app/pipes/DefaultNullOrEmptyPipe.spec.ts

```diff
@@ -31,30 +31,16 @@ describe('DefaultNullOrEmptyPipe', () => {
     expect(res).toEqual(expectedRes);
   });
 
-  it('should return date formatted string if value is a date string', () => {
-    const expectedRes = '14/02/2019';
-    const res = pipe.transform('2019-02-14T10:36:33.987Z');
-    expect(res).toEqual(expectedRes);
-  });
-
   it('should return value as default for type number', () => {
     const expectedRes = 3;
     const res = pipe.transform(3);
     expect(res).not.toBeNull();
     expect(res).toEqual(expectedRes);
   });
 
-  it('should check if date formatted string is same as value', () => {
-    const checkDate = new Date('2019-02-14T10:36:33.987Z');
-    const stringToCheck = '2019-02-14T10:36:33.987Z';
-    const res = pipe.transform(stringToCheck);
-    expect(checkDate.toISOString()).toEqual(stringToCheck);
-    expect(res).toEqual('14/02/2019');
-  });
-
-  it('a string of 2020-06-24 should be displayed in proper format', () => {
-    const stringToCheck = '2020-06-24';
-    const res = pipe.transform(stringToCheck);
-    expect(res).toEqual('24/06/2020');
+  it('should return expected string', () => {
+    const expectedRes = 'testing string';
+    const res = pipe.transform(expectedRes);
+    expect(res).toEqual('testing string');

```

Comment:
I think we need one more unit check that tests whether we return - for the else condition, also something minor but i think we could save a few lines if use a 'classic' if else, i don't think we need the switch case statement anymore